src/playbacks/hls/hls.js

```diff
@@ -119,6 +134,23 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal && this.recoverAttemptsRemaining > 0) {
+      this.recoverAttemptsRemaining -= 1
+      switch (data.type) {
+        case Hls.ErrorTypes.NETWORK_ERROR:
+          hls.startLoad()
+          break
+        case Hls.ErrorTypes.MEDIA_ERROR:
+          recover()
+          break
+        default:
```

Comment:
I think it's better to log this.
`Log.error(`hlsjs: this error could not be recovered, evt ${evt}, data ${data} `)`